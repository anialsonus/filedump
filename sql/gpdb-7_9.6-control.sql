-- control file
\! pg_filedump -c test/pg_control.gp7_9.6
