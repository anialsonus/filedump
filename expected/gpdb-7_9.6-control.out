-- control file
\! pg_filedump -c test/pg_control.gp7_9.6

*******************************************************************
* PostgreSQL File/Block Formatted Dump Utility
*
* File: test/pg_control.gp7_9.6
* Options used: -c
*******************************************************************

<pg_control Contents> *********************************************

                          CRC: Correct
           pg_control Version: 9600700
              Catalog Version: 301911081
            System Identifier: 6789867340620663399
                        State: IN PRODUCTION
                Last Mod Time: Wed Feb  5 16:07:39 2020
       Last Checkpoint Record: Log File (0) Offset (0x04f39d48)
   Previous Checkpoint Record: Log File (0) Offset (0x04f39c68)
  Last Checkpoint Record Redo: Log File (0) Offset (0x04f39d10)
          |-       TimeLineID: 1
          |-         Next XID: 0/548
          |-         Next OID: 24576
          |- Next Relfilenode: 10002
          |-       Next Multi: 1
          |-    Next MultiOff: 0
          |-             Time: Wed Feb  5 16:07:39 2020
       Minimum Recovery Point: Log File (0) Offset (0x00000000)
          Backup Start Record: Log File (0) Offset (0x00000000)
            Backup End Record: Log File (0) Offset (0x00000000)
End-of-Backup Record Required: no
       Maximum Data Alignment: 8
        Floating-Point Sample: 1234567
          Database Block Size: 32768
           Blocks Per Segment: 32768
              XLOG Block Size: 32768
            XLOG Segment Size: 67108864
    Maximum Identifier Length: 64
           Maximum Index Keys: 32
             TOAST Chunk Size: 8140

